### DISCUSSION TITLE
Shortcuts to Show/Hide Stashed Windows?

### DISCUSSION THREAD

**Discussion by helicos6** [rxhanson/RectanglePro-Community#201] [Q&A]
Is there any way we can use shortcuts to show/hide specific stashed windows?

**Comment by rxhanson** [comment]
Not at the moment. This is on my backlog for my next round of Stash features, though.

**Comment by helicos6** [comment]
How would it be implemented considering any number of windows can be stashed on each side of the screen?

**Reply by rxhanson** [reply]
I haven't thought through it entirely yet. There's a couple different thoughts I had:
1. Allow a shortcut for each side that cycles through the windows similarly to cycle stashed but for just one side (not precisely what you were looking for)
2. Arbitrarily apply numbers to each window that gets stashed and let the user configure modifier keys that would be pressed in combination with those numbers. Of course, this limits to 10 windows and it would likely only be usable in conjunction with color tabs where I can display the number on a tab.

Any other ideas?

**Reply by brucetm** [reply]
Number two is a usable idea I think! We really need to be able to unhide a stashed window with a key, though!!
